qhub/cli/_init.py

```diff
@@ -0,0 +1,496 @@
+            )
+        )
+
+    # GITOPS - REPOSITORY, CICD
+    rich.print(
+        (
+            "\n 🪴  This next section is [italic]optional[/italic] but recommended. If you wish to adopt a GitOps approach to managing this platform, "
+            "we will walk you through a set of questions to get that setup.\n\n"
+        )
+    )
+    if questionary.confirm(
+        "Would you like to adopt a GitOps approach to managing Nebari?",
+        default=False,
+        qmark=qmark,
+    ).ask():
+
+        repo_url = "http://{git_provider}/{org_name}/{repo_name}"
+
+        git_provider = questionary.select(
+            "Which git provider would you like to use?",
+            choices=["github.com", "gitlab.com"],
+            qmark=qmark,
+        ).ask()
+
+        org_name = questionary.text(
+            f"Which user or organization will this repository live under? ({repo_url.format(git_provider=git_provider, org_name='<org-name>', repo_name='')})",
+            qmark=qmark,
+        ).ask()
+
+        repo_name = questionary.text(
+            f"And what will the name of this repository be? ({repo_url.format(git_provider=git_provider, org_name=org_name, repo_name='<repo-name>')})",
+            qmark=qmark,
+        ).ask()
+
+        inputs.repository = repo_url.format(
+            git_provider=git_provider, org_name=org_name, repo_name=repo_name
+        )
+
+        if git_provider == "github.com":
+            inputs.repository_auto_provision = questionary.confirm(
+                f"Would you like the following git repository to be automatically created: {inputs.repository}?",
+                default=False,
+                qmark=qmark,
+            ).ask()
+
+        # TODO: create `check_repository_creds` function
+        if not disable_checks:
+            pass
+
+        if git_provider == "github.com":
+            inputs.ci_provider = CiEnum.github_actions.value.lower()
+        elif git_provider == "gitlab.com":
+            inputs.ci_provider = CiEnum.gitlab_ci.value.lower()
+
+    # SSL CERTIFICATE
+    rich.print(
+        (
+            "\n 🪴  This next section is [italic]optional[/italic] but recommended. If you want your Nebari domain to use a Let's Encrypt SSL certificate, "
+            "all we need is an email address from you.\n\n"
+        )
+    )
+    ssl_cert = questionary.confirm(
+        "Would you like to add a Let's Encrypt SSL certificate to your domain?",
+        default=False,
+        qmark=qmark,
+    ).ask()
+
+    if ssl_cert:
+        inputs.ssl_cert_email = questionary.text(
+            "Which email address should Let's Encrypt associate the certificate with?",
+            qmark=qmark,
+        ).ask()
+
+    # ADVANCED FEATURES
+    rich.print(
+        (
+            "\n 🪴  This next section is [italic]optional[/italic] and includes advanced configuration changes. "
```

Comment:
It be might nice to share what these advanced configs are so that users can make an informed decision about proceeding. Something like:

> includes advanced configuration changes like namespaces, terraform state, and Kubernetes versions.

We may also include a link to our docs, once we have a page for these :)

Reply:
Another good call! We definitely have the real estate to add more docs 🤘